priv/repo/migrations/20241023001501_rename_baseline_classifier_to_passthrough.exs

```diff
@@ -0,0 +1,20 @@
+defmodule Excision.Repo.Migrations.RenameBaselineClassifierToPassthrough do
+  use Ecto.Migration
+
+  import Ecto.Query
+  alias Excision.Repo
+
+  def change do
+    # fetches all classifiers with name "baseline" and renames them to "passthrough"
+    baseline_classifiers =
+      Excision.Excisions.Classifier
+      |> where(name: "baseline")
+      |> Repo.all()
+
+    for classifier <- baseline_classifiers do
+      classifier
+      |> Excision.Excisions.Classifier.changeset(%{name: "passthrough"})
+      |> Repo.update()
+    end
```

Comment:
Can' you do a bulk update here?  I guess since this product is so early, we don't have any customers so we won't have issues with slow migrations due to lots of data

Reply:
OK